packages/graphiql/example/schema.js

```diff
@@ -1,5 +1,5 @@
 /**
- *  Copyright (c) Facebook, Inc. and its affiliates.
+ *  Copyright (c) 2019 GraphQL Contributors and its affiliates.
```

Comment:
`and its affiliates` has started creeping through here, @acao

Reply:
ope!